src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -51,12 +53,84 @@ public sealed class NpgsqlRawCopyStream : Stream, ICancelable
 
         #region Constructor
 
-        internal NpgsqlRawCopyStream(NpgsqlConnector connector, string copyCommand)
+        private async Task SendQueryAsync(NpgsqlConnector connector, NpgsqlStatement statement, bool async)
```

Comment:
I think this would be common to all copy types (text, binary, raw binary...) so once again it makes more sense in NpgsqlCommand.